modules/0_git/test/main.cpp

```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```

Comment:
According to https://dkurt.github.io/openvino_practice/ you have already finished 1st task. Looks like the changes for OpenCV task are missed

Reply:
I was trying to send them. i will try again